content/04.results.md

```diff
@@ -14,15 +14,16 @@ Panel B shows the types and amounts of articles that we have used for analyses.
 
 ### Creation of an Annotated News Dataset
 
-We have analyzed the text and citations of 22,094 news-related articles hosted on "www.nature.com".
-The articles span 15 years from 2005 to 2020, and seven article types: Career Column, Career Feature, News, News and Views, News Feature, Technology Feature, and Toolbox.
+We have analyzed the text and citations of 16,080 news-related articles hosted on "www.nature.com".
```

Comment:
What do you think of making this the total number of frontmatter articles and including the span 15 years part here.

Reply:
yeah, I think this makes sense changed in https://github.com/greenelab/nature_news_manuscript/pull/3/commits/27be112764daab13da1f30c1027bf6d3dd3e9327